src/v0/destinations/ga4/networkHandler.js

```diff
@@ -38,39 +40,27 @@ const responseHandler = (destinationResponse, dest) => {
         validationCode,
         fieldPath
       } = response.validationMessages[0];
-      throw new ErrorBuilder()
-        .setStatus(400)
-        .setMessage(
-          `[GA4] Validation Server Response Handler:: Validation Error for ${dest} of field path :${fieldPath} | ${validationCode}-${description}`
-        )
-        .isTransformResponseFailure(true)
-        .setDestinationResponse(response?.validationMessages[0]?.description)
-        .setStatTags({
-          destType: dest,
-          stage: TRANSFORMER_METRIC.TRANSFORMER_STAGE.RESPONSE_TRANSFORM,
-          scope: TRANSFORMER_METRIC.MEASUREMENT_TYPE.API.SCOPE,
-          meta: getDynamicErrorType(status)
-        })
-        .build();
+      throw new NetworkError(
+        `Validation Server Response Handler:: Validation Error for ${dest} of field path :${fieldPath} | ${validationCode}-${description}`,
+        400,
+        {
+          [tags.TAG_NAMES.ERROR_TYPE]: getDynamicErrorType(status)
+        },
+        response?.validationMessages[0]?.description
```

Comment:
Should it directly be `destinationResponse`?

Reply:
Previously, in errorBuilder we were setting this only.